doc/radosgw/s3select.rst

```diff
@@ -69,11 +69,11 @@ Features Support
 +=================================+=================+=======================================================================+
 | Arithmetic operators            | ^ * % / + - ( ) | select (int(_1)+int(_2))*int(_9) from s3object;                       |
 +---------------------------------+-----------------+-----------------------------------------------------------------------+
-|                                 | ``%`` modulo    | select count(*) from s3object where cast(_1 as int)%2 == 0;           |
+|                                 | ``%`` modulo    | select count(*) from s3object where cast(_1 as int)%2 = 0;            |
 +---------------------------------+-----------------+-----------------------------------------------------------------------+
 |                                 | ``^`` power-of  | select cast(2^10 as int) from s3object;                               |
 +---------------------------------+-----------------+-----------------------------------------------------------------------+
-| Compare operators               | > < >= <= == != | select _1,_2 from s3object where (int(_1)+int(_3))>int(_5);           |
+| Compare operators               | > < >= <= = !=  | select _1,_2 from s3object where (int(_1)+int(_3))>int(_5);           |
 +---------------------------------+-----------------+-----------------------------------------------------------------------+
 | logical operator                | AND OR NOT      | select count(*) from s3object where not (int(1)>123 and int(_5)<200); |
```

Comment:
what is the difference between "int(_5)" and int(5)" (i.e., wrt the underscore);  I think I see these intermixed

Reply:
I've never seen _<number> in a sql dialect before--is that something aws s3select actually does?

Reply:
I assume these must be column names;  Is this ansi sql?  I've only seem aliases used for this

Reply:
> what is the difference between "int(_5)" and int(5)" (i.e., wrt the underscore); I think I see these intermixed

it's a mistake, it should be _{column positions)